environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - frictionless>=4.29,<5
   - isort
   - jupytext>=1.13,<2
+  - matplotlib>=3.5.0,<4
```

Comment:
You want `matplotlib-base` here. Otherwise, we pull in some desktop application for matplotlib.

Reply:
Adapted